db/version_set.cc

```diff
@@ -1744,13 +1746,24 @@ void VersionStorageInfo::ComputeExpiredTtlFiles(
   }
   const uint64_t current_time = static_cast<uint64_t>(_current_time);
 
-  for (int level = 0; level < num_levels() - 1; level++) {
+  uint64_t current_ttl = ttl;
+  for (int level = 0; level < num_levels(); level++) {
+    if (level == (num_levels() - 1)) {
```

Comment:
This mode works for dynamic leveling, but I don't know the impact for non-dynamic leveling.